test/sequential/test-http-strictmode.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
Certainly. I also just realized I didn't rename it to reflect the fact that `strictMode` isn't actually there. I'll make both changes now.